src/change_log.rb

```diff
@@ -53,6 +53,7 @@
   # defaults, go back 90 days
   options[:start_date] = Date.today - 90
   options[:end_date] = Date.today + 1
+  options[:branch] = 'develop-v3'
```

Comment:
users of the changelog script must now specify the branch they're generating the changelog for. E.g. to create a 3.x release we'd specify `develop-v3`, fora 2.x release we'd specify `develop-v2`